message config_values {
	required int32 led_brightness = 1;
}

